corehq/apps/case_migrations/views.py

```diff
@@ -54,7 +54,7 @@ def get_case_and_descendants(domain, case_id):
             if not c.closed]
 
 
-@mobile_auth
+@mobile_auth_or_formplayer
 def migration_restore(request, domain, case_id):
```

Comment:
@wpride I see that this uses the authenticated user to put in a registration element into the restore which contains the user's username and custom user data. Is that required? Just wondering how to handle it if there is no authed user.

Reply:
The username will default to `` if its not present and I think we can remove the `user_data` block entirely since its optional and won't have values for cases anyways